src/components/PurchaseOrder/PO.js

```diff
@@ -124,6 +131,26 @@ class PO extends Component {
     return false;
   }
 
+  closeOrder = (values) => {
+    const { mutator, resources, onCancel } = this.props;
+    const initialValues = get(resources, ['order', 'records', 0]);
```

Comment:
`initialValues` is a bad name. It's the prop name for redux form. You don't use it here, it's just the order